src/lib/logging/loggerRegistry.js

```diff
@@ -0,0 +1,43 @@
+const GLOBAL_LOGGERS = global.Box.PreviewLoggers || {};
+
+// Number of loggers that have been created since start of session.
+let loggerCount = 0;
+
+/**
+ * Add a logger instance to the global logger registry. Logger will be store as Logger<Number>.
+ * <Number> being the number of logs created this session.
+ *
+ * @param {Logger} logger - A logger to register
+ * @return {string} The name of the logger in the global registry.
+ */
+export function registerLogger(logger) {
```

Comment:
Did we decide on a plan for this?

Reply:
I think I'm going to close this PR, as how we're talking, the Logger will exist outside of Preview